annotations/lib/realm_property.dart

```diff
@@ -17,11 +17,30 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 class RealmModel {
-  const RealmModel();
+  final String? prefix;
+  final String? suffix;
```

Comment:
What's the reason for providing both a prefix and a suffix fields here? If I understand correctly, the intent is to enable generation of models with a name different from the one on which the `@RealmModel` annotation is applied. If that's the case, then isn't it more straightforward to provide a single field where the user can type whatever name they want?

Reply:
Hi @nirinchev 
These will both be removed. The idea was/is that optional prefix/suffix schema could be read from the build-yaml config file. I just slapped them in there for testing purposes. It is a bit early to review this PR.
To choose a specific name my current thinking is to re-use the `@MapTo` annotation.

Reply:
Fixed

Reply:
Sorry, didn't mean to rush you on this and I know it's draft - it's just a 3k+ LOC change and figured I can take a peek to make the actual review easier. Generally speaking though, in my reviews I'll mostly have questions and there's absolutely no expectation to take action on them immediately :)